cmd/dusk/server.go

```diff
@@ -244,16 +243,25 @@ func Setup() *Server {
 	// Creating the peer factory
 	readerFactory := peer.NewReaderFactory(processor)
 
+	// Create the listener and contact the voucher seeder
+	connector := peer.NewConnector(cfg.Get().Network.Port)
+
+	seeders := cfg.Get().Network.Seeder.Addresses
+	if len(seeders) != 0 {
+		if err := connector.Connect(seeders[0]); err != nil {
```

Comment:
if this is an array, would not make sense to iterate over it and try with more than the position zero ?